subgraph/schema.graphql

```diff
@@ -0,0 +1,233 @@
+type Account @entity {
```

Comment:
This file has been readed to the gitignore, but is still in the PR. Should be removed.